src/Grpc.Core.Api/Internal/CallDebuggerHelpers.cs

```diff
@@ -40,6 +50,25 @@ public static string DebuggerToString(AsyncCallState callState)
         return debugText;
     }
 
+    public static T? GetDebugValue<T>(AsyncCallState callState, string key) where T : class
```

Comment:
Why does `T` have to be a class?

Reply:
All the values so far as classes. It is easier to work with nullable values when constrained to class.